chrome/browser/android/adblock/adblock_bridge.cc

```diff
@@ -84,8 +84,39 @@ std::string replaceString(
     return subject;
 }
 
-std::string escapeSelector(std::string selector) {
-  return replaceString(replaceString(selector, "\\", "\\\\"), "\"", "\\\"");
+const char U2028_BYTES[] = { 0xE2, 0x80, 0xA8 };
```

Comment:
does it compile? I would expect that the cast is needed here, not on the next line.

Reply:
yes, i always compile before pushing

Reply:
I have just checked, it's not working, it can work because maybe chromium compiling scripts define `char` another way, could you please change `char` in the array to `uint8_t`?
```
const uint8_t U2028_BYTES[] = { 0xE2, 0x80, 0xA8 };
const std::string U2028 = reinterpret_cast<char const*>(U2028_BYTES);
```